packages/nimble-components/src/wafer-map/modules/experimental/worker-renderer.ts

```diff
@@ -1,14 +1,64 @@
+import { type Remote, transfer } from 'comlink';
 import type { WaferMap } from '../..';
 import { HoverDieOpacity } from '../../types';
+import { createMatrixRenderer } from '../create-matrix-renderer';
+import type { MatrixRenderer } from '../../../../build/generate-workers/dist/esm/source/matrix-renderer';
 
 /**
  * Responsible for drawing the dies inside the wafer map, adding dieText and scaling the canvas
  */
 export class WorkerRenderer {
+    private worker!: Remote<MatrixRenderer>;
+
+    private isWorkerAlive = false;
```

Comment:
I don't think we need this isWorkerAlive stuff anymore since we are tracking the async lifecycle better and waiting for tasks to finish before queueing new tasks. My goal is to not need it anymore

Reply:
indeed we no longer need that, I replaced the isWorkerAlive with a simple check if the worker is undefined, this is needed so we will not create new workers when we resize the wafer. https://github.com/ni/nimble/pull/1929/commits/8b6424051fe71d896b00acb65e9bc0021f637e22